packages/web-components/README.md

```diff
@@ -0,0 +1,87 @@
+# @trans-stat/auth-components
+Web Components built with FAST for easy user authentication UX.
+
+> NOTE: This package is still early in development. Expect breaking API changes often until the package reaches a stable 1.0.0 release.
+
+> NOTE: This library is meant to be used in conjuction with the backend packages published from [tas-security](https://github.com/trans-stat/tas-security).
+While these frontend Web Components can be used without the backend packages, they will work best as a fullstack solution.
+
+## installation
+### npm
+```shell
+npm install @trans-stat/auth-components
+```
+### yarn
+```shell
+yarn add @trans-stat/auth-components
+```
+
+## Usage
+### Configuring vscode intellisense for Web Components
+Add a `.vscode/settings.json` file to the root of your repository and set the html and css custom data paths to point to the auth-component's custom data files.
+```json
+{
+  "html.customData": [
+    "node_modules/@trans-stat/auth-components/dist/html-custom-data.json"
+  ],
+  "css.customData": [
+    "node_modules/@trans-stat/auth-components/dist/css-custom-data.json"
+  ]
+}
+```
+
+### `<biometric-card>`
```

Comment:
iValt doesn't just provide biometrics, so I wonder if the naming is accurate here? For example, goefencing could be one factor of the MFA, which is not biometrics or even date time values.

Reply:
I'm open to suggestions on naming. We could even just call this mfa-card and reuse it for other forms of multifactor.